organization.go

```diff
@@ -37,8 +37,8 @@ type OrganizationService struct {
 	client *Client
 }
 
-// GetOrganizationById finds an organization in zendesk by id
-func (s *OrganizationService) GetOrganizationById(organizationID string) (*Organization, *Response, error) {
+// GetOrganizationById finds an organization in Zendesk by ID
```

Comment:
comment on exported method OrganizationService.GetOrganizationByID should be of the form "GetOrganizationByID ..."